CHANGELOG.md

```diff
@@ -116,6 +116,30 @@ Other minor additions
 * Added new proof to `Data.Fin.Induction`:
   ```agda
   <-wellFounded : WellFounded _<_
+        ```
+
+* Added new properties to `Data.Fin.Properties`:
+  ```agda
+  toℕ≤n : ∀ {n} → (i : Fin n) → toℕ i ℕ.≤ n
```

Comment:
Thanks for adding the CHANGELOG entry! We tend to leave the initial implicit and explicit arguments out of the types to improve readability. You just need to give a flavour of the type. See some of the other entries for examples.